client/logic/ui/action_panel.gd

```diff
@@ -40,37 +35,37 @@ func _process(_delta: float) -> void:
 
 func set_buttons(actions: Array) -> void:
 	if GameConst.Actions.MOVE in actions:
-		($VBoxContainer/ButtonMove as Button).show()
```

Comment:
I'm allergic to this kind of stuff, can we do some kind of hash action => label/button and then we only need to do a loop from the hash? If you wish can be a ticket to avoid blocking this p/r

Reply:
fixed